### DISCUSSION TITLE
Meal Calories

### DISCUSSION THREAD

**Discussion by rayan589** [TandoorRecipes/recipes#1803] [Ideas]
Hello everyone,

I would like to suggest a new feature with each recipe, a kind of nutrition facts per serve for each meal. This is very important for people that calculate their macro and micronutrients, who exercise a lot or who are just curious.
Counting calories can help people get a better picture of what they eat each day and how much energy they are their your body.
It doesn't take much time, you can just google the nutrition facts of each ingredient in the meal (cooked ingredients tend to have different nutrition facts than when they are not) and add it all up.
Perhaps this will help to categorize meals into different diets (low carb, high calorie, high protein, super-carb, keto...)

Thank you !

**Comment by vabene1111** [comment]
hi, this is already planned with the #896 

i personally am not that well informed about the details of calorie/nutrition tracking so might need some input when i get to it but its on the way :)
